features/buyer/buyer_journey.feature

```diff
@@ -0,0 +1,26 @@
+@not-production @functional-test
+Feature: Buyer user journey through Digital Marketplace
```

Comment:
It could be useful to write that it's Buyer user's journey through GCloud, as on DOS it would be very different. I would also recommend including this in file-names, because now it's a bit confusing.